charts/refinery/templates/secretprovider.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.secretProvider.create -}}
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
```

Comment:
I'm not familiar with this type of resource.  Can it be installed into any k8s cluster?

Reply:
More information is available [here](https://secrets-store-csi-driver.sigs.k8s.io/), but yes I think any modern (>1.19) cluster should be able to support it.

Reply:
It looks like it does require a CustomResourceDefinition.  Normally a chart should but installable without external dependencies, but since this is optional I think it is ok.  Can you add in the values.yaml a comment linking to those docs and describing that the chart will only be installable with this feature enabled if those CRDs are present?

Reply:
Also in the readme table.

Reply:
Done!
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/1310969/236320864-8923e19f-6fe9-45c8-a916-be15ece9295d.png">